tests/plugins/button/manual/label.html

```diff
@@ -0,0 +1,28 @@
+<span id="results"></span>
+<div id="editor"></div>
+
+<script>
+	var results = CKEDITOR.document.getById( 'results' ),
+		editor = CKEDITOR.replace( 'editor', {
+			language: 'en',
+			on: {
+				instanceReady: function() {
+					var toolbox = editor.ui.space( 'toolbox' ),
+						langBtn = toolbox.findOne( '.cke_button' );
+
+					updateResults();
+
+					langBtn.on( 'click', updateResults );
```

Comment:
Clicking the button does not cover all cases:

* closing menu in any way (clicking elsewhere, selecting any option, pressing <kbd>Esc</kbd>…),
* opening menu via keyboard (kinda weird, because at least <kbd>Enter</kbd> should generate `click` event on link 🤔; yet there is always <kbd>Space</kbd>).

Not sure, but probably polling the value is the most bulletproof solution.